test/Grid.spec.tsx

```diff
@@ -393,8 +393,8 @@ describe('Grid', () => {
         element.dispatchEvent(new MouseEvent('dblclick', { bubbles: true }));
       }
 
-      function focusOut(element: Element) {
-        element.dispatchEvent(new FocusEvent('focusout', { bubbles: true }));
+      function focusOut(element: HTMLElement) {
+        element.blur();
```

Comment:
Could maybe fix grid-pro tests in a separate PR and backport to v24.5 in case it's also broken there.